packages/rps/src/redux/game-v2/__tests__/scenarios.ts

```diff
@@ -88,9 +88,9 @@ export const localStatesA = {
   weaponAndSaltChosen: s.weaponAndSaltChosen(propsA, salt),
   resultPlayAgain: s.resultPlayAgain(propsA, playerBWeapon, Result.YouWin),
   chooseWeapon2: s.chooseWeapon(propsA),
-  waitForRestart: s.waitForRestart(propsA),
-  shuttingDown: s.shuttingDown(propsA, 'InsufficientFundsOpponent'),
-  shuttingDownResign: s.shuttingDown(propsA, 'YouResigned'),
+  waitForRestart: s.waitForRestart(propsA, playerBWeapon, Result.YouWin),
+  shuttingDown: s.shuttingDown(propsA, playerBWeapon, Result.YouWin, 'InsufficientFundsOpponent'),
+  shuttingDownResign: s.shuttingDown(propsA, playerBWeapon, Result.Tie, 'YouResigned'),
```

Comment:
Should probably make this a completely different type of shutting down state. It's a bit nasty to have to put `Tie` in there as a default ..

Reply:
Can merge first though.

Reply:
Agreed. More recent changes show something less nasty, but still not the best solution.